src/lib/action.ts

```diff
@@ -65,7 +65,7 @@ const nameMap: PartialRecord<Action, string> = {
   addPageEnd: "+Page",
   selectAll: "Select All",
   duplicate: "Clone",
-  eraser: "Cut / Eraser",
+  eraser: "Erase",
```

Comment:
there's a pretty big reason why i wanted the label to be cut and erase rather than just erase? that is, it's not at all obvious that erasing something cuts it. so i'm going to revert this for now

Reply:
I feel that "eraser" decidedly refers to a tool, but "erase" feels like an action verb: i actually think "erase" is close in meaning to "cut" than "eraser," though it's pretty intuitive how the erase button would activate an eraser if there's nothing to immediately erase (cut). Also the existing form seems to have a part of speech disagreement.

(On a personal level, i don't care, but i do fine "erase" to be more natural. We also get the added benefit of easier styling)

Reply:
so, "cut" as in it erases it *and* puts it in your clipboard. like… like idk

Reply:
oh, i didn't realize it added to the clipboard. in that case idk

Reply:
what about just cut but with the eraser icon?

Reply:
then i wouldn't think it's reasonable to assume "cut" would turn on the eraser tool. especially since the erasing using the tool doesn't touch the clipboard

Reply:
hm. im still not sure what a good solution is

Reply:
In that case let's move it out of this PR and discuss in https://github.com/cjquines/qboard/issues/153